ROADMAP.md

```diff
@@ -20,18 +20,60 @@ Create radio drivers on raspberry pi and desktop ends which provide a programmab
 
 Generate CAR file using utility on raspberry pi. Use known file transfer protocol to transmit over radio interface to ground station. Ground station should receive, reassemble, and verify payload. Radio connection should be persistent and reliable.
 
```

Comment:
General... Are the 1st few POCs under MVP 0.0?   May want some indentation to improve readability

Reply:
Good idea, I added some grouping under milestones to improve readability.